src/toast/Toast.tsx

```diff
@@ -0,0 +1,55 @@
+import "./_toast.scss";
+
+import React, {useEffect} from "react";
+import ReactDOM from "react-dom";
+import classNames from "classnames";
+
+import {useToast} from "./util/toastHooks";
+
+function Toast() {
+  const {
+    toastState: {
+      isDisplayed,
+      data: {autoClose, timeout, content, mode, customClassName}
+    },
+    dispatchToastAction
+  } = useToast();
+  let toastRootNode = document.querySelector("#toast-root");
+
+  if (!toastRootNode) {
+    toastRootNode = document.createElement("div");
+    toastRootNode.setAttribute("id", "toast-root");
+  }
+
+  document.body.appendChild(toastRootNode);
```

Comment:
Shouldn't it be inside of the `if(!toastRootNode)` check?

Reply:
If doesn't exist a selector `#toast-root`, we are creating one inside of the `if(!toastRootNode)`, if already exists we just append it to the body. I think we should add for each two situations.

Reply:
I didn't get it. If it already exists why we need to append it?

Reply:
You are absolutely right. I don't remember what I thought 😅